code/components/gta-net-five/src/CloneManager.cpp

```diff
@@ -1510,6 +1513,11 @@ void CloneManagerLocal::HandleCloneSync(const char* data, size_t len)
 	net::Buffer netBuffer(reinterpret_cast<const uint8_t*>(data), len);
 	g_cloneMsgPacket = netBuffer.Clone();
 
+	if (len >= kSyncPacketWarnLength)
```

Comment:
This check should rather be warning on _send_ on the other side, since _receiving_ a large packet is the thing that's likely to fail.

Reply:
Could we keep the receive warning in addition to a send warning? I feel like it can't hurt to have information on both sides, especially once it's out in the wild and devs may want to get a warning when receiving large packets while they're investigating prior to any reproduction/isolation. For the issue this resolves, we weren't able to replicate it for some time and a receive warning would have done wonders.

Reply:
A receive warning wouldn't even be possible as an oversized packet (not even of this type - it's related to issues that may occur when fixing this) would simply fail to send/not arrive.